web/src/components/TopicCard.cy.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { TopicCard } from './TopicCard'
+import test_data from '../Tests/dummy_data.json'
+const dimensions = require('../Tests/dimensions.js')
+let topic= {...test_data.topic[0]}
+
+
+describe('General Tests',()=> {
```

Comment:
Very through testing of dimensions of all devices! Could this suite have a better name?
```suggestion
describe('Testing text alignment for each device dimension',()=> {
```

Reply:
Sounds good!